tests/gui/models/flash-state.spec.js

```diff
@@ -441,6 +470,52 @@ describe('Browser: flashState', function() {
 
     });
 
+    describe('.getFlashUuid()', function() {
+
+      const UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/;
+
+      it('should be initially undefined', function() {
```

Comment:
Presumably this is the _only_ time it'll ever be undefined?

EDIT: Nope, I've just seen that `ACTIONS.RESET_FLASH_STATE` does `.delete('flashUuid')`, but I've also just spotted that you _do_ check that with `it('should unset the flash uuid',` ;)